src/mono/mono/metadata/tabledefs.h

```diff
@@ -251,6 +251,7 @@ enum {
 #define PINVOKE_ATTRIBUTE_CALL_CONV_STDCALL   0x0300
 #define PINVOKE_ATTRIBUTE_CALL_CONV_THISCALL  0x0400
 #define PINVOKE_ATTRIBUTE_CALL_CONV_FASTCALL  0x0500
+#define PINVOKE_ATTRIBUTE_CALL_CONV_SWIFTCALL  0x0600
```

Comment:
these defines are supposed to match what's in ECMA-335 (or the augments). the swift call isn't going to be in there. so this is not needed.